packages/babel-minify-standalone/examples/example.htm

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
the file extension is missing `l` for `html`.

Reply:
It's consistent with babel-standalone: https://github.com/babel/babel/blob/7.0/packages/babel-standalone/examples/example.htm

Should I change both?

Reply:
Interesting, I thought this extension was dead long before. Its totally up to you :)